docs/test/async.md

```diff
@@ -16,17 +16,41 @@ test('name', 'must be unique', new Promise((resolve, reject) => {
             } else {
                 resolve(); // completes. doesn't mark the test as failing
             }
-        }
+        });
 }));
 ```
 
 ## Returning a Promise / Async/Await
 
 ```js
 test('name',  'Should be unique', async () => {
-    const res = await doesUserExist(user)
+    const res = await doesUserExist(user);
     return res;
 });
 
 test('name',  'I fail', async () => Promise.reject());
+```
+
+## Rejecting with rejection message
+
+What if your promise can reject with different messages? No problem!
+You can reject the promise with your own message by passing it to the
+rejection callback.
+
+Notice that when using rejection messages we do not pass a `statement`
```

Comment:
That's not true according to your implementation. The correct wording according to your implementation is that "we do not *need* to pass statement", but we could, and it will be used as a fallback.